diode-server/netboxdiodeplugin/client.go

```diff
@@ -42,6 +43,11 @@ const (
 	defaultBaseURL = "http://127.0.0.1:8080/api/plugins/diode"
 
 	defaultHTTPTimeoutSeconds = 5
+
+	// NetboxBranchHeader is an HTTP header that indicates the netbox branch to target
+	NetboxBranchHeader = "X-Netbox-Branch"
+	// NetboxBranchParam is a query parameter that indicates the netbox branch to target
+	NetboxBranchParam = "_branch"
```

Comment:
Let's try to keep consitency with `NetBox` name casing when possible, also it might matter in the header as well as should match https://github.com/netboxlabs/netbox-branching/blob/develop/netbox_branching/constants.py#L8

```suggestion
	// NetBoxBranchHeader is an HTTP header that indicates the NetBox branch to target
	NetBoxBranchHeader = "X-NetBox-Branch"
	// NetBoxBranchParam is a query parameter that indicates the NetBox branch to target
	NetBoxBranchParam = "_branch"
```